examples/gatsby-builder-plugin-example/plugins/gatsby-plugin-builder/src/transforms.js

```diff
@@ -0,0 +1,62 @@
+const { GraphQLObjectType, GraphQLNonNull } = require(`gatsby/graphql`);
```

Comment:
can we just use the graphql plugin in this example and not have to include all of this stuff?